workshops/Tkinter/README.md

```diff
@@ -0,0 +1,188 @@
+
+#### Finally the code will look like this.
+
+```python
+import tkinter as tk
+def fun1():
+  F1=tk.Frame(window,bg="light salmon",width=800,height=560).place(x=0,y=0)
+  tk.Label(F1,text="Creativity \nis \nintelligence \nhaving \nFun.",font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").pack()
+  tk.Button(F1,command=fun2,text="Next",bg="aqua",fg="black",width=20).place(x=612,y=530)
+def fun2():
+  F2=tk.Frame(window,bg="light salmon",width=800,height=560).place(x=0,y=0)
+  tk.Label(F2,text="If the plan \ndoesn’t work, \nchange the plan,\n but never the \ngoal.",font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").place(x=200,y=100)
+  tk.Button(F2,command=fun3,text="Next",bg="aqua",fg="black",width=20).place(x=612,y=530)
+def fun3():
+  F3=tk.Frame(window,bg="light salmon",width=800,height=560).place(x=0,y=0)
+  tk.Label(F3,text="Avoiding failure\n is to avoid \nprogress.",font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").place(x=200,y=100)
+  tk.Button(F3,command=fun4,text="Next",bg="aqua",fg="black",width=20).place(x=612,y=530)
+def fun4():
+  F4=tk.Frame(window,bg="light salmon",width=800,height=560).place(x=0,y=0)
+  tk.Label(F4,text="Sometimes\n you win,\n sometimes \nyou learn.",font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").place(x=240,y=100)
+  tk.Button(F4,command=fun5,text="Next",bg="aqua",fg="black",width=20).place(x=612,y=530)
+def fun5():
+  F5=tk.Frame(window,bg="light salmon",width=800,height=560).place(x=0,y=0)
+  tk.Label(F5,text="Instead of wondering when your \nnext vacation is, maybe you \nshould set up a life you don’t \nneed to escape from.",font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").place(x=30,y=100)
+window = tk.Tk()
+window.minsize(800, 560)
+window.title("Simple UI")
+window.config(bg="black")
+tk.Label(window, font=("Helvetica", 60, "bold"),text="QUOTES HUB", bg="black", fg="white").pack()
+tk.Button(window,command=fun1,text="Get In",bg="aqua",fg="black",width=20).place(x=315,y=300)
+window.mainloop()
+
+```
+
+Its simple just go through the functions and workflow.
+
+![ezgif com-gif-to-mp4](https://cloud-5m2nwfs8r.vercel.app/ezgif.com-video-to-gif-4.gif)
+
+## Step 4: Your Custom Part
+
+Now you know Tkinter a little bit. You should not stop here to learn more and master in UIs you need to practice a lot. The best way to learn it is to look at the world and think of an idea then try to make it.
+
+## Lastly don't get stuck.
+
+I am attaching the Tkinter documentation take a look at [Learn Tkinter](https://docs.python.org/3/library/tk.html).
+Also if you are new to python take a look at [python](https://www.python.org/doc/) documentation too
+
+### Softwares made using Tkinter by other hackers.
+
+[Anupriya Shaji](https://repl.it/@Anupriya567/qtshb#main.py)
+
+[Kk Haridev](https://repl.it/@DandaThor/harri-kkkkk#main.py)
+
+[Bezlin Johnson](https://repl.it/@bezlin/graphics#main.py)
```

Comment:
How is this demo different from what the person just made?

These demos all need to be adaptations of the project